File: AkitaModel/CultureInfo.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Skyware.Lis.AkitaModel;

public class CultureInfo
{

    public static IFormatProvider InvariantCulture { get; internal set; }

    public int Id { get; set; }

    public string CurrencySymbol { get; set; }

    public int CurrencyPosition { get; set; }

    public string CurrencyDecSep { get; set; }

    public string CurrencyGroupSep { get; set; }

    public int CurrencyDecDigits { get; set; }

    public string NumberDecSep { get; set; }

    public string NumberGroupSep { get; set; }

}

File: AkitaModelJava/akitamodel/src/main/java/skyware/lis/akitamodel/CultureInfo.java
package skyware.lis.akitamodel;

import lombok.Data;


@Data
public class CultureInfo {

    private int id;

    private String currencySymbol;

    private int currencyPosition;

    private String currencyDecSep;

    private String currencyGroupSep;

    private int currencyDecDigits;

    private String numberDecSep;

    private String numberGroupSep;

}
